content/1-introduction-1-intro.md

```diff
@@ -1 +1,84 @@
 # Introduction
+
+\todo{This might be too high-level, so make it more focused. Also add an early hint of what I'll do, which is not that broad. Remember to start easy (but rich) before going to more complex things.}
+
+Current scientific knowledge describes the complexity of life, and of human life in particular, through a wide array of theoretical and empirical approaches.
```

Comment:
Ah oui, là je suis catégorique – c'est l'équivalent de "De tout temps, les h[H]ommes ont (...)" de la dissert de philo ;)  Il faut commencer par un truc lié à ta thèse, pas à 42.  L'idéal étant de commencer par une anecdote super bien trouvée mais, là, personnellement, je n'ai jamais été très inspiré pour ça... si tu y arrives, kudos ! :)  Sinon, la mise en contexte suffit.
Tiens, je te copie-colle le début de l'intro de ma thèse:
<<Agents producing, manipulating, exchanging knowledge are forming as a whole a socio- semantic complex system: a complex system made of agents who work on and are influenced by semantic content, by flows of information in which they are fully immerged but, at the same time, on which they can have an impact and leave their footprints. Social psychologists and epistemologists, inter alia, have already a long history in studying the properties of such knowledge communities. Yet, the massive availability of informational content and the potential for extensive interactivity has made the focus slip from single “groups of knowledge” to the entire “society of knowledge”. (...)>> 
Et je te re-copie-colle le début de BCP:
<<The reciprocal influence between cognition and culture has a long history in both social science and psychology. While this question has been the subject of intense debate in the social sciences in the 20th century, today’s discussion is mostly structured by proponents from cognitive science, who construe culture as an evolutionary process analogous and parallel to biological evolution. (...)>>